Delivered-To: mine260309@gmail.com
Received: by 2002:a4f:7f0c:0:0:0:0:0 with SMTP id a12csp60865ivd;
        Thu, 3 Jan 2019 13:57:12 -0800 (PST)
X-Google-Smtp-Source: ALg8bN4YIRFisw+fFnz0Yfyfg2HgWHadUubDQdC074WSHPe2/HG66f9VwbHqZ0Pqflw/9lyOlazx
X-Received: by 2002:a17:902:82c2:: with SMTP id u2mr48483846plz.110.1546552631981;
        Thu, 03 Jan 2019 13:57:11 -0800 (PST)
ARC-Seal: i=1; a=rsa-sha256; t=1546552631; cv=none;
        d=google.com; s=arc-20160816;
        b=K6Gs5C/9kpAlPdG2rWkPa/cgNXIzPD9JRNg/ENODhdo3y7yqygI5ABrMwyVTxmjYrz
         wdtcQdR+mX8Lwl/SSd7DA9JrDaKdymgtHQeJrWo8Sg2EuNoW4jRQ7Hr3vifwJc3DoTO7
         CCJkg8sT8P6GSBp6VUIdsyuntcFjhEN/YQ4E+6q0qXHlmZ8lUuPQ9QHcG5W+PxDMRpL3
         Fc8lIy4KRUDv0nwEbsQniy1IKEDGD3nLx9UoHiSbYcoQXiN3XjODgk3u/j+reKSX11lz
         g6TsGjswUoTSVUgpgAt9bbwQdEftHs1nu6yn1TlKUNwIKWFKEfJSjIdqz/s9kvguteVH
         Nz9g==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=google.com; s=arc-20160816;
        h=sender:errors-to:cc:list-subscribe:list-help:list-post:list-archive
         :list-unsubscribe:list-id:precedence:message-id:references
         :in-reply-to:date:subject:to:from:delivered-to;
        bh=9RL4at2YG0JYs9Gp9rOaIBXVZGn01K4b2bGvNUvRzmw=;
        b=hrAg6wrqH1AIJJF/fVPei1s5uhahxuaz2po+TPFSuQ7u8iBLHfgrwyaXaW5mZjR4Pd
         28dGUQ5hkgNpon3GGwIWdKaNs/MhYAtBAjKUqxYirngCAP/CcfoAo8+d2NyuA57VnjxA
         P1OG2bfCibtJoY5UjU4J812CqtMZLW0C+q1pv5nevXhNIdNsFRIvxLOqHPunxAJZcgkV
         Y5iQRljeo/r7ligniJliGov3Bg05PJ4BvShEn8hEEsoaN+JvfImbxgHI1NdO7aaUxsWk
         DwrCl/GAd+SGaOzijjH1tu2JO7jvaSAiWIhAcM88/IfpBUPDjkFWUCeXbFX00AMkaGNv
         KKxQ==
ARC-Authentication-Results: i=1; mx.google.com;
       spf=pass (google.com: domain of openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org designates 2401:3900:2:1::3 as permitted sender) smtp.mailfrom="openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org";
       dmarc=fail (p=NONE sp=NONE dis=NONE) header.from=ibm.com
Return-Path: <openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org>
Received: from lists.ozlabs.org (lists.ozlabs.org. [2401:3900:2:1::3])
        by mx.google.com with ESMTPS id k22si49162836pgl.29.2019.01.03.13.57.11
        for <mine260309@gmail.com>
        (version=TLS1_2 cipher=ECDHE-RSA-CHACHA20-POLY1305 bits=256/256);
        Thu, 03 Jan 2019 13:57:11 -0800 (PST)
Received-SPF: pass (google.com: domain of openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org designates 2401:3900:2:1::3 as permitted sender) client-ip=2401:3900:2:1::3;
Authentication-Results: mx.google.com;
       spf=pass (google.com: domain of openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org designates 2401:3900:2:1::3 as permitted sender) smtp.mailfrom="openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org";
       dmarc=fail (p=NONE sp=NONE dis=NONE) header.from=ibm.com
Received: from lists.ozlabs.org (lists.ozlabs.org [IPv6:2401:3900:2:1::3])
	by lists.ozlabs.org (Postfix) with ESMTP id 43W1wj3vLHzDqLq
	for <mine260309@gmail.com>; Fri,  4 Jan 2019 08:57:09 +1100 (AEDT)
X-Original-To: openbmc@lists.ozlabs.org
Delivered-To: openbmc@lists.ozlabs.org
Authentication-Results: lists.ozlabs.org;
 spf=pass (mailfrom) smtp.mailfrom=linux.ibm.com
 (client-ip=148.163.156.1; helo=mx0a-001b2d01.pphosted.com;
 envelope-from=eajames@linux.ibm.com; receiver=<UNKNOWN>)
Authentication-Results: lists.ozlabs.org;
 dmarc=none (p=none dis=none) header.from=linux.ibm.com
Received: from mx0a-001b2d01.pphosted.com (mx0a-001b2d01.pphosted.com
 [148.163.156.1])
 (using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
 (No client certificate requested)
 by lists.ozlabs.org (Postfix) with ESMTPS id 43W1sc541LzDqLw
 for <openbmc@lists.ozlabs.org>; Fri,  4 Jan 2019 08:54:28 +1100 (AEDT)
Received: from pps.filterd (m0098394.ppops.net [127.0.0.1])
 by mx0a-001b2d01.pphosted.com (8.16.0.22/8.16.0.22) with SMTP id
 x03LsOes114352
 for <openbmc@lists.ozlabs.org>; Thu, 3 Jan 2019 16:54:26 -0500
Received: from e33.co.us.ibm.com (e33.co.us.ibm.com [32.97.110.151])
 by mx0a-001b2d01.pphosted.com with ESMTP id 2psrhvnfys-1
 (version=TLSv1.2 cipher=AES256-GCM-SHA384 bits=256 verify=NOT)
 for <openbmc@lists.ozlabs.org>; Thu, 03 Jan 2019 16:54:25 -0500
Received: from localhost
 by e33.co.us.ibm.com with IBM ESMTP SMTP Gateway: Authorized Use Only!
 Violators will be prosecuted
 for <openbmc@lists.ozlabs.org> from <eajames@linux.ibm.com>;
 Thu, 3 Jan 2019 21:54:24 -0000
Received: from b03cxnp08026.gho.boulder.ibm.com (9.17.130.18)
 by e33.co.us.ibm.com (192.168.1.133) with IBM ESMTP SMTP Gateway: Authorized
 Use Only! Violators will be prosecuted; 
 (version=TLSv1/SSLv3 cipher=AES256-GCM-SHA384 bits=256/256)
 Thu, 3 Jan 2019 21:54:21 -0000
Received: from b03ledav003.gho.boulder.ibm.com
 (b03ledav003.gho.boulder.ibm.com [9.17.130.234])
 by b03cxnp08026.gho.boulder.ibm.com (8.14.9/8.14.9/NCO v10.0) with ESMTP id
 x03LsKXM16842908
 (version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-GCM-SHA384 bits=256 verify=FAIL);
 Thu, 3 Jan 2019 21:54:20 GMT
Received: from b03ledav003.gho.boulder.ibm.com (unknown [127.0.0.1])
 by IMSVA (Postfix) with ESMTP id 79A406A04D;
 Thu,  3 Jan 2019 21:54:20 +0000 (GMT)
Received: from b03ledav003.gho.boulder.ibm.com (unknown [127.0.0.1])
 by IMSVA (Postfix) with ESMTP id 1F9226A047;
 Thu,  3 Jan 2019 21:54:20 +0000 (GMT)
Received: from talon7.ibm.com (unknown [9.41.179.222])
 by b03ledav003.gho.boulder.ibm.com (Postfix) with ESMTP;
 Thu,  3 Jan 2019 21:54:19 +0000 (GMT)
From: Eddie James <eajames@linux.ibm.com>
To: openbmc@lists.ozlabs.org
Subject: [PATCH dev-4.19 5/6] media: dt-bindings: media: Add Aspeed Video
 Engine binding documentation
Date: Thu,  3 Jan 2019 15:54:07 -0600
X-Mailer: git-send-email 1.8.3.1
In-Reply-To: <1546552448-22621-1-git-send-email-eajames@linux.ibm.com>
References: <1546552448-22621-1-git-send-email-eajames@linux.ibm.com>
X-TM-AS-GCONF: 00
x-cbid: 19010321-0036-0000-0000-00000A747355
X-IBM-SpamModules-Scores: 
X-IBM-SpamModules-Versions: BY=3.00010339; HX=3.00000242; KW=3.00000007;
 PH=3.00000004; SC=3.00000272; SDB=6.01141310; UDB=6.00593988; IPR=6.00921536; 
 MB=3.00024957; MTD=3.00000008; XFM=3.00000015; UTC=2019-01-03 21:54:23
X-IBM-AV-DETECTION: SAVI=unused REMOTE=unused XFE=unused
x-cbparentid: 19010321-0037-0000-0000-00004A34C8E9
Message-Id: <1546552448-22621-6-git-send-email-eajames@linux.ibm.com>
X-Proofpoint-Virus-Version: vendor=fsecure engine=2.50.10434:, ,
 definitions=2019-01-03_10:, , signatures=0
X-Proofpoint-Spam-Details: rule=outbound_notspam policy=outbound score=0
 priorityscore=1501
 malwarescore=0 suspectscore=1 phishscore=0 bulkscore=0 spamscore=0
 clxscore=1015 lowpriorityscore=0 mlxscore=0 impostorscore=0
 mlxlogscore=999 adultscore=0 classifier=spam adjust=0 reason=mlx
 scancount=1 engine=8.0.1-1810050000 definitions=main-1901030187
X-BeenThere: openbmc@lists.ozlabs.org
X-Mailman-Version: 2.1.29
Precedence: list
List-Id: Development list for OpenBMC <openbmc.lists.ozlabs.org>
List-Unsubscribe: <https://lists.ozlabs.org/options/openbmc>,
 <mailto:openbmc-request@lists.ozlabs.org?subject=unsubscribe>
List-Archive: <http://lists.ozlabs.org/pipermail/openbmc/>
List-Post: <mailto:openbmc@lists.ozlabs.org>
List-Help: <mailto:openbmc-request@lists.ozlabs.org?subject=help>
List-Subscribe: <https://lists.ozlabs.org/listinfo/openbmc>,
 <mailto:openbmc-request@lists.ozlabs.org?subject=subscribe>
Cc: Hans Verkuil <hverkuil-cisco@xs4all.nl>,
 Mauro Carvalho Chehab <mchehab+samsung@kernel.org>,
 Eddie James <eajames@linux.ibm.com>
Errors-To: openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org
Sender: "openbmc" <openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org>

Document the bindings.

Signed-off-by: Eddie James <eajames@linux.ibm.com>
Reviewed-by: Rob Herring <robh@kernel.org>
Signed-off-by: Hans Verkuil <hverkuil-cisco@xs4all.nl>
Signed-off-by: Mauro Carvalho Chehab <mchehab+samsung@kernel.org>
---
 .../devicetree/bindings/media/aspeed-video.txt     | 26 ++++++++++++++++++++++
 1 file changed, 26 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/media/aspeed-video.txt

diff --git a/Documentation/devicetree/bindings/media/aspeed-video.txt b/Documentation/devicetree/bindings/media/aspeed-video.txt
new file mode 100644
index 0000000..78b464a
--- /dev/null
+++ b/Documentation/devicetree/bindings/media/aspeed-video.txt
@@ -0,0 +1,26 @@
+* Device tree bindings for Aspeed Video Engine
+
+The Video Engine (VE) embedded in the Aspeed AST2400 and AST2500 SOCs can
+capture and compress video data from digital or analog sources.
+
+Required properties:
+ - compatible:		"aspeed,ast2400-video-engine" or
+			"aspeed,ast2500-video-engine"
+ - reg:			contains the offset and length of the VE memory region
+ - clocks:		clock specifiers for the syscon clocks associated with
+			the VE (ordering must match the clock-names property)
+ - clock-names:		"vclk" and "eclk"
+ - resets:		reset specifier for the syscon reset associated with
+			the VE
+ - interrupts:		the interrupt associated with the VE on this platform
+
+Example:
+
+video-engine@1e700000 {
+    compatible = "aspeed,ast2500-video-engine";
+    reg = <0x1e700000 0x20000>;
+    clocks = <&syscon ASPEED_CLK_GATE_VCLK>, <&syscon ASPEED_CLK_GATE_ECLK>;
+    clock-names = "vclk", "eclk";
+    resets = <&syscon ASPEED_RESET_VIDEO>;
+    interrupts = <7>;
+};
-- 
1.8.3.1

